F21/smedipally/.py

```diff
@@ -0,0 +1,95 @@
+def LowerCaseConverter(word,lang):
+
```

Comment:
Can we follow the universal style guide for the line alignment

Reply:
Yes, We can! But for better understanding of code i did it on purpose!